src/R/Components/Impl/Documentation/Commands/OpenDocumentationCommand.cs

```diff
@@ -0,0 +1,53 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License. See LICENSE in the project root for license information.
+
+using System.Diagnostics;
+using System.IO;
+using System.Threading.Tasks;
+using Microsoft.R.Components.Controller;
+using Microsoft.R.Components.InteractiveWorkflow;
+
+namespace Microsoft.R.Components.Documentation.Commands {
+    public sealed class OpenDocumentationCommand : IAsyncCommand {
+        private readonly IRInteractiveWorkflow _interactiveWorkflow;
+        private readonly string _onlineUrl;
+        private readonly string _localRelativePath;
+
+        public OpenDocumentationCommand(IRInteractiveWorkflow interactiveWorkflow, string onlineUrl, string localRelativePath = null) {
+            _interactiveWorkflow = interactiveWorkflow;
+            _onlineUrl = onlineUrl;
+            _localRelativePath = localRelativePath;
+        }
+
+        public CommandStatus Status {
+            get {
+                return CommandStatus.Supported | CommandStatus.Enabled;
+            }
+        }
+
+        public Task<CommandResult> InvokeAsync() {
+            var url = _onlineUrl;
+
+            if (!string.IsNullOrEmpty(_localRelativePath)) {
```

Comment:
Exactly. Online version should be good enough for this relatively less common case.